package.json

```diff
@@ -45,6 +45,7 @@
       }
     ],
     "okapiInterfaces": {
+      "browse": "0.6 1.0",
       "inventory": "10.10 11.0 12.0",
```

Comment:
Shouldn't you add 13.0 for inventory.

Reply:
This PR is only about "Browse". The rest part relates to https://issues.folio.org/browse/UIIN-2288

Reply:
ok Thank you